Sources/ProjectDescription/Arguments.swift

```diff
@@ -2,14 +2,30 @@ import Foundation
 
 /// A collection of arguments and environment variables.
 public struct Arguments: Equatable, Codable {
-    public let environment: [String: String]
+    public let environmentVariables: [String: EnvironmentVariable]
     public let launchArguments: [LaunchArgument]
 
+    @available(*, deprecated, message: "please use environmentVariables instead")
     public init(
-        environment: [String: String] = [:],
```

Comment:
This seems to be a breaking change. I'd keep this as it is with the deprecated warning, and then we'll remove it when we have the opportunity to do a major release.

Reply:
@lo1tuma please add back the default value 🙏

Reply:
Good catch. There are multiple initializers, and this only affects calling `Arguments()` without any parameters. I’ve added the default value to second initializer to bring this functionality back and not causing the deprecation warning to be triggered for calling `Arguments()`.